cypress/page_objects/createBoard.js

```diff
@@ -0,0 +1,40 @@
+export default class CreateBoard {
```

Comment:
you dont need to export class twice, currently you do it on start with export default class and at the end of the class you export const again